Storybook support!!

DanyElAlgo opened the pull request:
Yeah

DanyElAlgo committed:
Storybook support!!

Yeah

RaidenMT commented on OrderNow-React/src/components/Button/Button.css:
No usaremos clases de css. Vamos a utilizar Tailwind. Solo por esta ocasión se aprobará este archivo pero el botón se debe modificar en el siguiente PR

RaidenMT commented on OrderNow-React/src/components/Button/index.js:
Eliminar archivo, no es necesario. Se debe adjuntar el export en el mismo archivo Button.jsx

RaidenMT commented on OrderNow-React/src/components/input-field/index.js:
Eliminar archivo, no es necesario. Se debe adjuntar el export en el mismo archivo Button.jsx

RaidenMT commented on OrderNow-React/src/components/input-field/input-field.css:
Se aprueba solo por esta ocasión, pero ya ir mirando las clases de Tailwind

RaidenMT commented on OrderNow-React/src/components/submit-button/index.js:
Eliminar archivo, no es necesario. Se debe adjuntar el export en el mismo archivo submitButton.jsx

RaidenMT commented on OrderNow-React/src/components/input-field/input-field.jsx:
Arreglar nomenclatura del nombre. submit-button e input-field están en minúsculas y separan las palabras por guion. En cambio, Button está en mayúscula, cuál se va a mantener?

RaidenMT commented on OrderNow-React/src/components/submit-button/submit-button.css:
Solo se aprobará por esta ocasión, luego modificar por clases de Tailwind

RaidenMT commented on OrderNow-React/vitest.workspace.js:
Esto qué es?

RaidenMT commented:
Hacer pull antes de subir por favor

DanyElAlgo committed:
Creación de tarjetas de productos realizado

La instalación de tailwind fue exitosa, poco a poco se irán implementando más historias para utilizar
